netci.groovy

```diff
@@ -125,7 +125,7 @@ def CreateLinuxBuildTasks = { machine, configTag, linuxBranch, nonDefaultTaskSet
             def infoScript = 'bash jenkins/get_system_info.sh'
             def debugFlag = buildType == 'debug' ? '--debug' : ''
             def buildScript = "bash ./build.sh -j=`nproc` ${debugFlag} --cxx=/usr/bin/clang++-3.8 --cc=/usr/bin/clang-3.8"
-            def testScript = "bash test/runtests.sh"
+            def testScript = "bash test/runtests.py Basics/hello.js"
```

Comment:
I think it is better to keep the `runtests.sh` script and add any tests you want to run to that script as we go (presumably it will grow?). It is difficult to test changes to the netci.groovy script as I noted in another comment. For tests which will change often, it is best to keep the information about which tests to run in another file, like `runtests.sh`. Additionally, this way we don't need the build definitions tied to the specific way we run tests and we are free to change them later.